ext/rotoscope/rotoscope.c

```diff
@@ -8,16 +8,18 @@
 #include <sys/file.h>
 
 #include "rotoscope.h"
+#include "tracepoint.h"
```

Comment:
I took the `rs_tracepoint_t` definition out of `rotoscope.h` so it could be used in `stack.h` as well without a recursive inclusion.